### PR TITLE
Fixing bad argument passing for data observatory metadata

### LINKED ISSUE #11318: DO measures for points only show US and France
If you add a DO analysis to a **points** node, you can only choose US or France

image

And once there, you can only select a subset of all the available measures:

image

image

**This behavior doesn't affect polygon nodes.**

cc @ethervoid

### PR DESCRIPTION
**Pull request by ethervoid**
Closes #11318 

Lecacybuildermatada function should receive no argument when is not a polygon geometry type

### CODE REVIEW

File: lib/assets/core/javascripts/cartodb3/editor/layers/layer-content-views/analyses/data-observatory-metadata.js [hunk 1]
```diff
@@ -3,7 +3,14 @@ var Backbone = require('backbone');
 var cdb = require('cartodb.js');
 
 function getQueryForGeometryType (geometryType) {
-  return 'select * from obs_legacybuildermetadata(\'' + (geometryType === 'polygon' ? 'sum' : '') + '\')';
+  var QUERY = 'select * from obs_legacybuildermetadata({{aggregate_type}})';
+
```
Comment (rafatower) [inline lib/assets/core/javascripts/cartodb3/editor/layers/layer-content-views/analyses/data-observatory-metadata.js]: the linter is complaining about trailing spaces

Comment (ethervoid) [inline lib/assets/core/javascripts/cartodb3/editor/layers/layer-content-views/analyses/data-observatory-metadata.js]: Trailing spaces removed

**Comment by ethervoid** [comment]
Changed it to make it clearer

**Comment by javisantana** [comment]
could we merge this as soon as we can and then add a automated test

**Pull request merged**